sys/Makefile

```diff
@@ -139,6 +139,9 @@ endif
 ifneq (,$(filter cord_ep,$(USEMODULE)))
     DIRS += net/application_layer/cord/ep
 endif
+ifneq (,$(filter riotboot_%,$(USEMODULE)))
```

Comment:
This needs to point to `riotboot` only instead. I'll add another fixup for this. @kaspar030 do you agree?